docs/serving/spec/knative-api-specification-1.0.md

```diff
@@ -1315,22 +1353,22 @@ Restrictions to the values of the field are noted in the Description column.
    </td>
    <td>int
 <br>
-(Required)
+(Optional)
 <br>
 Min: 0
 <br>
 Max: 100
    </td>
    <td>The
 
-<a href="#request-routing">percentage of requests which should be allocated from
+<a href="#request-routing">Percent is optionally used to specify the percentage of requests which should be allocated from
```

Comment:
This doesn't make it clear how to select percentage based routing (which is the default, I think). Maybe phrase this as "percentage based routing is the default unless selected by an extension"?

Reply:
I think I cover that now below.